yt_dlp/extractor/spreaker.py

```diff
@@ -91,34 +96,38 @@ class SpreakerIE(InfoExtractor):
     }, {
         'url': 'https://api.spreaker.com/v2/episodes/12534508?export=episode_segments',
         'only_matching': True,
+    }, {
+        'note': 'episode',
+        'url': 'https://www.spreaker.com/episode/grunge-music-origins-the-raw-sound-that-defined-a-generation--60269615',
+        'info_dict': {
+            'id': '60269615',
+            'display_id': 'grunge-music-origins-the-raw-sound-that-',
+            'ext': 'mp3',
+            'title': 'Grunge Music Origins - The Raw Sound that Defined a Generation',
+            'description': str,
+            'timestamp': 1717468905,
+            'upload_date': '20170809',
+            'uploader': 'Katie Brown 2',
+            'uploader_id': '17733249',
+            'duration': 1063.42,
+            'view_count': int,
+            'like_count': int,
+            'comment_count': int,
+            'series': '90s Grunge',
+        },
     }]
 
     def _real_extract(self, url):
         episode_id = self._match_id(url)
+        query = {}
+        if key := traverse_obj(url, ({parse_qs}, 'key', 0)):
+            query = {'key': key}
         data = self._download_json(
             f'https://api.spreaker.com/v2/episodes/{episode_id}',
-            episode_id)['response']['episode']
+            episode_id, query=query)['response']['episode']
```

Comment:
could do this and remove the added lines above
```suggestion
            episode_id, query=traverse_obj(parse_qs(url), {'key': ('key', 0)}))['response']['episode']
```